src/functions-and-arrays.js

```diff
@@ -1,42 +1,99 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+
+function maxOfTwoNumbers( n1, n2) {
+  if (n1>n2) {
+    return n1;
+  }  else if (n2>n1) {
+     return n2;
+  } else {
+    return n1,n2;
```

Comment:
Here you can just return one of the arguments (n1 or n2) as they have the same value.